Doc/library/pydoc.rst

```diff
@@ -33,13 +33,13 @@ as text on the console.  The same text documentation can also be viewed from
 outside the Python interpreter by running :program:`pydoc` as a script at the
 operating system's command prompt. For example, running ::
 
-   <python you are using> -m pydoc sys
+   <python> -m pydoc sys
```

Comment:
Please remove the angled brackets. So just `python -m pydoc sys`.

Reply:
yes